(docs) Add description and keywords (#206)
openebs/website: docs/versioned_docs/version-2.11.0/additional-info/faq.md

---
id: faqs
title: OpenEBS FAQ
keywords: 
 - OpenEBS FAQ
 - FAQs

> description: The FAQ section for a website helps to address common concerns, questions, and objections that users have.

Reviewer: ```suggestion
description: The FAQ section about OpenEBS helps to address common concerns, questions, and objections that users have about OpenEBS.
```
